### ISSUE TITLE
[Spacer] Doesn't seem to be getting parsed?

### ISSUE THREAD

**Issue by seansmyth** [foundation/foundation-emails#484]
**How can we reproduce this bug?**
Add spacer INKY code in a template. In the outputted generated HTML file, the `<spacer size="100"></spacer>` doesn't get parsed. As far as I'm aware, I'm up to date - see screenshots so I'm unsure what the issue might be.

```
<p>Stuff on top</p>
<spacer size="100"></spacer>
<p>Stuff on bottom</p>
```

**What did you expect to happen?**
The 

`<spacer size="100"></spacer>`

to be parsed into the below - like the docs

```
<table class="spacer">
  <tbody>
    <tr>
      <td height="100px" style="font-size:100px;line-height:100px;">&#xA0;</td>
    </tr>
  </tbody>
</table>
```

**What happened instead?**
`<spacer size="100"></spacer>`  remains in the outputted HTML file.

![screen-shot-2016-06-17-at-11 00 30](https://cloud.githubusercontent.com/assets/7364220/16147549/96a0c19c-347b-11e6-8293-34da50790be1.gif)
![screen shot 2016-06-17 at 11 01 31](https://cloud.githubusercontent.com/assets/7364220/16147547/969da5f2-347b-11e6-9809-5ba796098001.png)
![screen shot 2016-06-17 at 11 01 21](https://cloud.githubusercontent.com/assets/7364220/16147548/969e0114-347b-11e6-85d0-255d63482fd4.png)

**Comment by rafibomb** [comment]
This feature was released in 2.1 so you'd need at least that version to get it. The version of the CLI does not affect this.

If you package.json is pointed to 2.1.0 https://github.com/zurb/foundation-emails-template/blob/master/package.json#L16

The run `npm update`

**Comment by rafibomb** [comment]
We'll close this here as it's working in new 2.1.0 projects and the docs - thanks!
